examples/enrollmentrequest.yaml

```
@@ -1,34 +1,14 @@
-    MmY5NTdmYWE3NDJmYzcwNTAwMjY3NDhiYzc5MDY1Yzg4MTlkMWIwMFkwEwYHKoZI
-    zj0CAQYIKoZIzj0DAQcDQgAEHBMC30NRJTJ3NWOlei9YWkBM/MzoNDGyMRKuzE0/
-    cKkyXVJexHq0xIn22YCNN3ZDNxxwW6LlQg3ijGnvStQOb6AAMAoGCCqGSM49BAMC
-    A0gAMEUCIQDSrxi6krdCKcLChdMB3w7IqmDlZhREGbHMCESVn6ssPAIgZL+QrJtw
-    8R+r4piaYvZktrsk3yCIuz3hI7loYmXP3d8=
+apiVersion: api.flightctl.io/v1alpha1  
+kind: EnrollmentRequest  
+metadata:  
+  name: 13d632d86373caaeda64da4052f957faa742fc7050026748bc79065c8819d1b0  
+spec:  
+  csr: |  
+    -----BEGIN CERTIFICATE REQUEST-----  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)